example/centos/centos-9-aarch64-wsl.yaml

```diff
@@ -0,0 +1,131 @@
+otk.version: "1"
+
+otk.define:
+  packages:
+    build:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: aarch64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/aarch64.yaml"
+        packages:
+          include:
+            - coreutils
+            - glibc
+            - platform-python
+            - python3
+            - python3-iniparse
+            - rpm
+            - selinux-policy-targeted
+            - systemd
+            - tar
+            - xz
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: aarch64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/aarch64.yaml"
+        packages:
+          include:
+            - alternatives
+            - audit-libs
+            - basesystem
+            - bash
+            - ca-certificates
+            - coreutils-single
+            - crypto-policies-scripts
+            - curl-minimal
+            - dejavu-sans-fonts
+            - dnf
+            - filesystem
+            - findutils
+            - gdb-gdbserver
+            - glibc-minimal-langpack
+            - gmp
+            - gnupg2
+            - gobject-introspection
+            - hostname
+            - langpacks-en
+            - libcurl-minimal
+            - openssl
+            - pam
+            - passwd
+            - procps-ng
+            - python3
+            - python3-inotify
+            - redhat-release
+            - rootfiles
+            - rpm
+            - sed
+            - setup
+            - shadow-utils
+            - subscription-manager
+            - systemd
+            - tar
+            - tpm2-tss
+            - tzdata
+            - util-linux
+            - vim-minimal
+            - yum
+          exclude:
+            - gawk-all-langpacks
+            - glibc-gconv-extra
+            - glibc-langpack-en
+            - openssl-pkcs11
+            - python-unversioned-command
+            - redhat-release-eula
+            - rpm-plugin-systemd-inhibit
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: pipeline/build/tar.yaml
+    - name: os
+      build: name:build
+      stages:
+        - otk.external.osbuild-make-depsolve-dnf4-rpm-stage:
+            packageset: ${packages.os}
+            gpgkeys:
+              otk.include: "common/gpgkeys.yaml"
+        - type: org.osbuild.fix-bls
+          options: {}
+        - type: org.osbuild.locale
+          options:
+            language: en_US.UTF-8
+        - type: org.osbuild.timezone
+          options:
+            zone: America/New_York
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
```

Comment:
This is curious, we don't seem to install a kernel but we list it here in sysconfig. Obviously not an issue with this PR as it just replicates images but might still be worth adding a comment (here and in the other file) that we should look into this. I assume this will be the basis of new e.g. centos-10 images where we can look into dropping this kernel option.

Reply:
I think this might be an issue in images.  We add the kernel sysconfig to all image types unconditionally across a distro as part of the default image config.  We should probably have some conditions for that.

Reply:
Thanks! If it's an issue with the "images" lib I think it would be good to add a comment here (well, a comment here would be good also if it's fine, I find it confusing otherwise to have a kernel reference here without a kernel :)

Reply:
What do you want the comment to say?

Reply:
I've added the comment verbatim (with typos fixed ;)), but only to the added wsl omnifest here.

Reply:
We want the comment for both files (both arches), no? Or will one file go away RSN with the dedup work?

Reply:
I've added a duplicated comment to the `x86_64` omnifest; I'm not sure how long it will take for this to go away :)